file-formats/edck/type/zif_aff_edck_v1.intf.abap

```diff
@@ -0,0 +1,72 @@
+INTERFACE zif_aff_edck_v1
+  PUBLIC.
+
+  TYPES ty_check_classification TYPE c LENGTH 1.
+  TYPES ty_check_category TYPE c LENGTH 6.
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Check Category</p>
+      "! Consistency check category
+      "! $values {@link zif_aff_edck_v1.data:co_check_category}
+      "! $default {@link if_aff_edck_v1.data:co_check_category.exist}
```

Comment:
```suggestion
      "! $default {@link zif_aff_edck_v1.data:co_check_category.existence_check}
```
This should fix the schema issue from the GitHub action.

Reply:
For the default value of the Check Category, the `exist` also needs to be changed to `existence_check` since that is the name of the variable.